packages/fluentui/react-northstar-fela-renderer/test/felaSanitizeCssPlugin-test.ts

```diff
@@ -63,16 +62,6 @@ describe('felaSanitizeCssPlugin', () => {
     });
   });
 
-  test('should skip excluded CSS props', () => {
-    const style = {
-      display: 'block',
-      margin: '0 0 0 0',
-      propertyWithInvalidValue: 'rgba(',
-    };
-
-    expect(sanitize(style, { skip: ['propertyWithInvalidValue'] })).toEqual(style);
```

Comment:
This feature was removed to simplify the returned signature. Was never used anywhere.